src/_locales/vi.config

```diff
@@ -0,0 +1,183 @@
+@extension_description
+Chế độ tối cho mọi trang web. Chăm sóc đôi mắt của bạn, sử dụng chủ đề tối cho trình duyệt ban đêm và hàng ngày.
+
+@loading_please_wait
+Đang tải vui lòng đợi...
+
+
+#==== Top section ====
+
+@on
+Bật
+
+@off
+Tắt
+
+@toggle_current_site
+Chuyển đổi trang web này 
```

Comment:
Remove the whitesapce after `này` So it pass the checks :)

Reply:
Highlighted text I mean with the whitespace 
![](https://i.imgur.com/R9ymT4E.png)

Reply:
You can continue editing your Pull Request. But if there's no other mistakes I will fix it myself.